staff.go

```diff
@@ -23,17 +25,57 @@ func (s StaffApiServiceImpl) DeleteAuction(ctx context.Context, s2 string) (open
 	panic("implement me")
 }
 
-func (s StaffApiServiceImpl) Deleteclass(ctx context.Context, s2 string) (openapi.ImplResponse, error) {
+func (s *StaffApiServiceImpl) Deleteclass(ctx context.Context, query openapi.RequestUser) (openapi.ImplResponse, error) {
 	//TODO implement me
 	panic("implement me")
 }
 
-func (s StaffApiServiceImpl) EditClass(ctx context.Context, body openapi.ClassesClassBody) (openapi.ImplResponse, error) {
-	//TODO implement me
-	panic("implement me")
+func (a *StaffApiServiceImpl) EditClass(ctx context.Context, body openapi.RequestEditClass) (openapi.ImplResponse, error) {
```

Comment:
move all staff functions to another file

Reply:
I don't understand what to move where?
If staff functions are not in staff.go then where should I put them?

Reply:
I don't understand what to move where?
If staff functions are not in staff.go then where should I put them?

Reply:
> I don't understand what to move where? If staff functions are not in staff.go then where should I put them?

staffimpl.go?